.github/workflows/release-beta.yml

```
@@ -0,0 +1,42 @@
+on:
+  push:
+    branches:
+      - feat_v3.x:
+  pull_request:
+    branches:
+      - feat_v3.x
+
+
+jobs:
+  release-beta:
+    if: startsWith(github.event.head_commit.message, 'chore(release): ') && contains(github.event.head_commit.message, '-beta.')
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 14: mapping values are not allowed in this context

(syntax-check)